DQMOffline/Trigger/plugins/TopMonitor.cc

```diff
@@ -0,0 +1,541 @@
+#include "DQMOffline/Trigger/plugins/TopMonitor.h"
```

Comment:
In principle top dependent variables can be added. I would keep it as it is for the moment...